java-client/src/main/java/org/opensearch/client/opensearch/indices/PutMappingRequest.java

```diff
@@ -293,26 +170,6 @@ public final Time masterTimeout() {
 		return this.masterTimeout;
 	}
 
-	/**
-	 * Specify timeout for connection to cluster-manager
-	 * <p>
-	 * API name: {@code cluster_manager_timeout}
-	 */
-	@Nullable
-	public final Time clusterManagerTimeout() {
```

Comment:
ClusterManagerTimeout is part of the inclusive naming #166, so we can keep that.